cmd/ddev/cmd/root.go

```diff
@@ -22,8 +22,12 @@ var (
 	plugin   = "local"
 	// 1 week
 	updateInterval = time.Hour * 24 * 7
+	siteName       string
```

Comment:
It feels like we should be working to reduce/eliminate our global vars. I'm not sure it adds much here. Any objection to just removing it?